### ISSUE TITLE
Cannot read private member from an object whose class did not declare it

### ISSUE THREAD

**Issue by benoitdelorme** [darkroomengineering/lenis#300]
**Describe the bug**
I use Lenis in my [Nuxt3-Lenis module](https://github.com/benoitdelorme/nuxt3-lenis) and since I updated with the latest version of Lenis I have an error when I'm using lenis.start() and lenis.stop(). It worked with version 1.0.27

<img width="581" alt="Capture d’écran, le 2024-02-15 à 08 03 45" src="https://github.com/studio-freight/lenis/assets/561485/7055d1e7-523b-4d6e-ae57-b60a5f3ddc13">

I tried to fix it but without much success.
```
start() {
  this.isStopped = false //  this.#isStopped = false
  this.reset()
}

stop() {
  this.isStopped = true //  this.#isStopped = true
  this.animate.stop()

  this.reset()
}
```

**Comment by clementroche** [comment]
You mean calling `lenis.start()` throws this error ? Can you provide a codesandbox ? It seems you're trying to change a private property of Lenis.

It might comes from how you use Lenis in your implementation since it works as expected here https://codesandbox.io/p/sandbox/small-cdn-yk9cqy?file=%2Fsrc%2FApp.js%3A8%2C16

**Comment by plcdnl** [comment]
i have the same problem. It happens when i call the functions start() and stop()

**Comment by plcdnl** [comment]
this is the sandbox: https://codesandbox.io/p/live/4825f9e6-107a-408c-8a57-aa5a1e22db81

**Comment by clementroche** [comment]
I can't access your codesandbox @plcdnl 

![image](https://github.com/studio-freight/lenis/assets/28361254/bb24a756-62d4-4b77-ba84-d7e40c5fe7f4)

**Comment by thiervoj** [comment]
I'm facing the same issue when calling `start()` and `stop()`. It seems to happen from Lenis v1.0.35, because v1.0.34 works fine on my end.

I think it comes from [this commit](https://github.com/darkroomengineering/lenis/commit/14654b9d47d05c2f02387dd6581fe2c6eef4ed6f#diff-2c2f75feaffb03271a017ad69a593ca8d6ad7e1d68e4c6b6102cfc1c2e350da6R44), where private properties have been introduced.

According to the [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties#private_fields) about Private Fields, I would suggest accessing these private class properties using `this.#isStopped` instead of `this.isStopped` for example : https://github.com/darkroomengineering/lenis/blob/main/packages/lenis/src/index.ts#L276

(I'm using Nuxt 3.10.2)

**Comment by clementroche** [comment]
Why is this happening only with nuxt ?

**Comment by thiervoj** [comment]
Good question, have you tested it in a vanilla js environment ?

**Comment by plcdnl** [comment]
> Why is this happening only with nuxt ?

https://codesandbox.io/p/devbox/lenis-nuxt-bug-tfdzlp

here the sandbox. Sorry!

**Comment by clementroche** [comment]
https://codesandbox.io/p/sandbox/small-cdn-yk9cqy?file=%2Fsrc%2FApp.js%3A8%2C16 @thiervoj

**Comment by thiervoj** [comment]
I can't access your sandbox either @clementroche

![image](https://github.com/darkroomengineering/lenis/assets/10621317/439ce6e1-b038-49cb-a5f7-6fadea40d86d)

**Comment by clementroche** [comment]
It seems like codesandbox make draft private by default now 

here is @thiervoj  : https://codesandbox.io/p/sandbox/small-cdn-yk9cqy?file=%2Fsrc%2FApp.js

**Comment by clementroche** [comment]
can you try outside of defineNuxtPlugin ?

**Comment by plcdnl** [comment]
> can you try outside of defineNuxtPlugin ?

https://codesandbox.io/p/devbox/lenis-nuxt-bug-forked-9kkc6c?

**Comment by plcdnl** [comment]
Ok found, it happens with vue reactive/refs. With simple vars it works without problems

**Comment by clementroche** [comment]
ok that's not ideal anyway, thank you for your feedback, I must find a better way to handle it imo. Imma revert to previous way.

**Comment by plcdnl** [comment]
Thank you for your work!

**Comment by benoit-laradev** [comment]
Thanks you!

**Comment by benoitdelorme** [comment]
Thank you!

**Comment by clementroche** [comment]
It's fixed in v1.0.37
